test/functional-tests/test.sh

```diff
@@ -11,29 +14,28 @@ function tmp_dunstrc {
 }
 
 function start_dunst {
-        killall dunst
-        ../../dunst -config $1 &
-        sleep 0.1
+        killall dunst 2>/dev/null
+        $DUNST -config $1 &
+        sleep 0.05
 }
 
 function basic_notifications {
-    ../../dunstify -a "dunst tester"         "normal"    "<i>italic body</i>"
-    ../../dunstify -a "dunst tester"  -u c   "critical"   "<b>bold body</b>"
-    ../../dunstify -a "dunst tester"         "long body"  "This is a notification with a very long body"
-    ../../dunstify -a "dunst tester"         "duplucate"
-    ../../dunstify -a "dunst tester"         "duplucate"
-    ../../dunstify -a "dunst tester"         "duplucate"
-    ../../dunstify -a "dunst tester"         "url"        "www.google.de"
+    $DUNSTIFY -a "dunst tester"         "normal"    "<i>italic body</i>"
+    $DUNSTIFY -a "dunst tester"  -u c   "critical"   "<b>bold body</b>"
+    $DUNSTIFY -a "dunst tester"         "long body"  "This is a notification with a very long body"
+    $DUNSTIFY -a "dunst tester"         "duplicate"
+    $DUNSTIFY -a "dunst tester"         "duplicate"
+    $DUNSTIFY -a "dunst tester"         "duplicate"
+    $DUNSTIFY -a "dunst tester"         "url"        "www.google.de"
 
 }
 
 function show_age {
     echo "###################################"
     echo "show age"
     echo "###################################"
-    killall dunst
-    ../../dunst -config dunstrc.show_age &
-    ../../dunstify -a "dunst tester"  -u c "Show Age" "These should print their age after 2 seconds"
+    start_dunst dunstrc.show_age &
```

Comment:
Missed it last time ...

When you're putting this command in background you're basically nullifying the sleep in that function, which could potentially cause issues when start up is slowed down.

Reply:
I thought this was the previous behavior. I'll check again. 

Anyhow is the graphical part working now?

Reply:
It was just an oversight